src/store/indexeddb-remote-backend.js

```diff
@@ -31,20 +31,16 @@ import Promise from 'bluebird';
 const RemoteIndexedDBStoreBackend = function RemoteIndexedDBStoreBackend(
     workerScript, dbName, WorkerApi,
 ) {
+    this._workerScript = workerScript;
     this._dbName = dbName;
-    this._worker = new WorkerApi(workerScript);
+    this._workerApi = WorkerApi;
```

Comment:
i don't understand why we're assigning this to a field rather than just using it direct when we need it